seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -1644,7 +1650,7 @@ def test_filtered_get_es_variants(self):
     def test_sv_get_es_variants(self):
         setup_responses()
         search_model = VariantSearch.objects.create(search={
-            'annotations': {'structural': ['DUP']},
+            'annotations': {'new_structural_variants': ['NEW']},
```

Comment:
Use `NEW_SV_FIELD` instead of `'new_structural_variants'`?

Reply:
why? we only hardcode constant in tests when we use them in many places and in this test file we barely use constants for the search dictionaries and this wasn't a constant before?

Reply:
That must be fine for a test file. I didn't notice it.